File: ios/MobilePassSDK/Model/LogItem.swift
//
//  LogItem.swift
//  MobilePassSDK
//
//  Created by Erinc Cakir on 19.08.2021.
//

import Foundation

public struct LogItem {
    public var level:      Int
    public var code:       Int?
    public var message:    String
    public var time:       Date
    
    init(level: Int, code: Int?, message: String) {
        self.level      = level
        self.code       = code
        self.message    = message
        self.time       = Date()
    }
}

File: android/mobilepasssdk/src/main/java/com/armongate/mobilepasssdk/model/LogItem.java
package com.armongate.mobilepasssdk.model;

import java.util.Date;

public class LogItem {
    public int      level;
    public Integer  code;
    public String   message;
    public Date     time;

    public LogItem(int level, Integer code, String message) {
        this.level      = level;
        this.code       = code;
        this.message    = message;
        this.time       = new Date();
    }
}
